src/actix/api/issues_api.rs

```diff
@@ -17,7 +17,20 @@ async fn get_issues(ActixAccess(access): ActixAccess) -> impl Responder {
     process_response(response, timing)
 }
 
+#[post("/issues/clear")]
+async fn clear_issues(ActixAccess(access): ActixAccess) -> impl Responder {
+    let timing = Instant::now();
+    let response = access
+        .check_global_access(AccessRequirements::new().manage())
+        .map(|_| {
+            issues::clear();
+            true
+        });
+    process_response(response, timing)
+}
```

Comment:
Could be simplified as:
```suggestion
fn clear_issues(ActixAccess(access): ActixAccess) -> impl Future<Output = HttpResponse> {
    crate::actix::helpers::time(async move {
        access.check_global_access(AccessRequirements::new().manage())?;
        issues::clear();
        Ok(true)
    })
}
```

Reply:
Nice! looks cleaner